packages/component/src/providers/internal/SendBox/SendBoxComposer.tsx

```diff
@@ -116,10 +132,33 @@ const SendBoxComposer = ({ children }: PropsWithChildren<{}>) => {
         ]) as readonly [Timeout, Timeout];
       } else {
         scrollToEndRef.current?.();
-        apiSubmitSendBox();
+        if (combineAttachmentsAndText && files.length) {
```

Comment:
The old logic use Redux to send, while the new logic use the hook directly. This is inconsistent.